<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instagram</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>little_insta.png</normaloff>little_insta.png</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semilight</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Instagram</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="midLineWidth">
     <number>-1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_line">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semilight</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Yu Gothic UI Semilight&quot;;
border-style: outset;
border-radius: 10px;
border-width: 1px;
min-width: 3em;
padding: 6px;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="find_user">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>106</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Yu Gothic UI Semilight&quot;;
color: rgb(0, 170, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(0, 170, 255);
min-width: 3em;
padding: 6px;</string>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>106</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Yu Gothic UI Semilight&quot;;
color: rgb(0, 170, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(0, 170, 255);
min-width: 3em;
padding: 6px;</string>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QLabel" name="main_photo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Фото</string>
    </property>
   </widget>
   <widget class="QLabel" name="username_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLabel" name="stats">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semilight</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Статы</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>901</width>
      <height>450</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 0px;
border-color: rgb(255, 255, 255);
border-style: outset;</string>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="rowCount">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>300</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>300</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>300</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>300</number>
    </attribute>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="add_photo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Yu Gothic UI Semilight&quot;;
color: rgb(0, 170, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(0, 170, 255);
min-width: 3em;
padding: 6px;</string>
    </property>
    <property name="text">
     <string>Добавить фото</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_profile">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Yu Gothic UI Semilight&quot;;
color: rgb(0, 170, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(0, 170, 255);
min-width: 3em;
padding: 6px;</string>
    </property>
    <property name="text">
     <string>Редактировать профиль</string>
    </property>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sub">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Yu Gothic UI Semilight&quot;;
color: rgb(0, 170, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: rgb(0, 170, 255);
min-width: 3em;
padding: 6px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Подписаться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="username">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Yu Gothic UI Semilight&quot;;
border-style: outset;
border-width: 0px;
border-radius: 10px;
min-width: 3em;
padding: 6px;
background-color: rgb(244, 244, 244);</string>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
